package-lock.json

```diff
@@ -1,8 +1,10906 @@
 {
   "name": "edu-client",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "edu-client",
+      "version": "0.0.1",
+      "license": "MIT",
+      "dependencies": {
```

Comment:
Please dont add unwanted lock file changes. Just those changes are needed where package is installed.

What you should do is just copy the lock file as it is from the main repo and do npm install again. It will just change the required lines in lock file.

Reply:
@Abhishek-kumar09 restored the original package-lock.json